src/cli.rs

```diff
@@ -57,6 +57,10 @@ w - match full words only
     */
     pub flags: Option<String>,
 
+    #[arg(short = 'l', long = "line-buffered", default_value_t = false)]
```

Comment:
```suggestion
    #[arg(short, long, default_value_t)]
```

Nit: we can use `clap`'s "magic values" for all of these